configs/config/benchmark/linear_image_classification/datasets/cifar10.yaml

```diff
@@ -0,0 +1,47 @@
+# @package _global_
```

Comment:
for all the config files, I am fine if we want to move them out to `linear_image_classification/<dataset_name>/.yaml` as it seems like these could be benchmarks that can be referenced to at least one paper?

Reply:
As discussed together, the best for now is to leave these configuration in <datasets> for as long as I am not able to get the exact results published in the papers (I am taking CLIP as reference here), so that the message is "we support those, but these are not part of the benchmark yet".